dotCMS/src/main/java/com/dotcms/publisher/business/PublisherAPIImpl.java

```diff
@@ -10,6 +10,7 @@
 
 import com.dotcms.system.event.local.business.LocalSystemEventsAPI;
 import com.dotcms.system.event.local.type.pushpublish.AddedToQueueEvent;
+import org.mapdb.DB;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.mapdb.DB'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)